salt/modules/linux_sysctl.py

```diff
@@ -2,22 +2,21 @@
 '''
 Module for viewing and modifying sysctl parameters
 '''
-from __future__ import absolute_import, print_function, unicode_literals
+from __future__ import absolute_import
 
 # Import python libs
 import logging
 import os
 import re
-import string
 
 # Import salt libs
-from salt.ext import six
+import salt.ext.six as six
+import salt.utils
 from salt.ext.six import string_types
 from salt.exceptions import CommandExecutionError
-import salt.utils.data
-import salt.utils.files
+from salt.utils.odict import OrderedDict
 import salt.utils.systemd
-import salt.utils.stringutils
+import string
```

Comment:
Why is this import being moved from above? That should stay within the "Python Imports" list.